vllm/sequence.py

```diff
@@ -834,6 +856,10 @@ def token_chunk_size(self) -> int:
         assert self._token_chunk_size is not None
         return self._token_chunk_size
 
+    def finish_step(self) -> None:
```

Comment:
not used in this PR. Will it be used in the next PR with a scheduler?

Reply:
yes it will be used in AsyncLLMEngine